packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -1180,183 +1231,203 @@
+								</SelectContent>
+							</Select>
+							<Input
+								type="text"
+								placeholder="Custom font"
+								onChange={(e) => onUpdate({ font: e.target.value })}
+							/>
+						</div>
+						<Input
+							type="number"
+							value={fontSize}
+							onChange={(e) => onUpdate({ fontSize: parseInt(e.target.value) })}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)